test/k8sT/istio.go

```diff
@@ -36,27 +36,31 @@ var _ = Describe("K8sIstioTest", func() {
 		// installed.
 		istioSystemNamespace = "istio-system"
 
-		istioVersion = "1.7.6"
+		istioVersion = "1.8.2"
 
 		// Modifiers for pre-release testing, normally empty
-		prerelease     = "" // "-beta.1"
-		istioctlParams = ""
+		prerelease     = "-beta.1"
+		istioctlParams = // ""
+		" --set values.pilot.image=docker.io/jrajahalme/istio_pilot:1.8.2-beta.1" +
+			" --set values.global.proxy.image=docker.io/jrajahalme/istio_proxy:1.8.2-beta.1" +
```

Comment:
Should we move these to `docker.io/cilium-dev` or `quay.io/cilium-dev` just like other product images?

Reply:
I should have made this a draft, sorry. I'll push the release images once CI passes, in order to avoid releasing faulty images.